tests/models/interfaces/test_interfaces.py

```diff
@@ -13,10 +19,22 @@ class TestModelInterfaces(unittest.TestCase):
     @test_utils.skip_if_macos
     def test_mmbt_hm_interface(self):
         model = MMBT.from_pretrained("mmbt.hateful_memes.images")
+        self._test_model_performance(model)
+
+    @test_utils.skip_if_no_network
+    @test_utils.skip_if_windows
+    @test_utils.skip_if_macos
+    def test_mmbt_hm_interface_from_file(self):
+        with tempfile.NamedTemporaryFile(suffix=".pth") as tmp:
+            self._create_checkpoint_file(tmp.name)
+
+            model = MMBT.from_pretrained(tmp.name, model_interface=True)
```

Comment:
This is a non-backwards compatible change, we need to discuss this. In MMBT, can you specifically set `model_interface` to `True` by default?